hack/cluster-deploy.sh

```diff
@@ -85,7 +85,12 @@ until _kubectl -n kubevirt get kv kubevirt; do
     sleep 1
 done
 
-# wait until KubeVirt is ready
-_kubectl wait -n kubevirt kv kubevirt --for condition=Available --timeout 6m || (echo "KubeVirt not ready in time" && dump_kubevirt && exit 1)
+# Wait until KubeVirt is ready
+count=0
+until _kubectl wait -n kubevirt kv kubevirt --for condition=Available --timeout 5m; do
```

Comment:
Why don't we just either increase the timeout value here or use an env variable?

Reply:
The problem is that increasing timeout wont help, because there are real hiccups such as (all those i saw in this case)

Unable to connect to the server: EOF
Unable to connect to the server: net/http: TLS handshake timeout
request timed out

they are temporary in our case, so just had to sleep, let the cluster relax and retry the command and it worked

Reply:
Or asked the other way round, did you observe that kubevirt got into ready state after more that 5 minutes? When and where did this happen?

Reply:
what we see is the following:
the original code gets one of the 3 errors above
and the make cluster-sync "fail"

now if you continue to try the wait manually or just oc get pods -A
if will give errors those errors for around 1 minutes, after 1 minute, 
all commands will back to normal and triggering the wait command again will success

Reply:
OK then. Thanks for your answers!